nearlib/test/integration.test.js

```diff
@@ -117,11 +117,29 @@ describe('with deployed contract', () => {
             setArgs);
         expect(scheduleResult.hash).not.toBeFalsy();
         await nearjs.waitForTransactionResult(scheduleResult);
-        const secondViewFunctionResult = await nearjs.callViewFunction(
+        const getValueResult = await nearjs.callViewFunction(
             contractName,
             'getValue', // this is the function defined in hello.wasm file that we are calling
             {});
-        expect(secondViewFunctionResult).toEqual(setCallValue);
+        expect(getValueResult).toEqual(setCallValue);
+
+        // test that load contract works and we can make calls afterwards
```

Comment:
why not make a separate test case?

Reply:
I don't have a strong opinion either way. It's roughly the same amount of code

Reply:
See e.g. `'can get logs from method result'` test